### ISSUE TITLE
Highlighting not readable for nested headlines

### ISSUE THREAD

**Issue by jdittrich** [mangecoeur/document-outline#30]
**Story:** As a user, I want to know where my current position is in the document (-outline) 

**Problem:** The highlight showing the position in the document outline is very dark for nested headlines with a depth of 1, so that the text is not readable anymore. (assumes standard theme *atom light*, for e.g. base16 it works without problems, since the highlight just is a bordered box there)

**Possible solution**: 
* Highlighting using an underline, or,
* Adjusting the opacity of the text accordingly, so it is always by an amount darker than the highlight (assuming light theme, vice versa for dark)
* Highlight using white with opacity (resulting in shades of lighter grays) (assuming light theme, vice versa for dark)

**Comment by bmikol** [comment]
+1 for this issue. I'm currently working on a 3-deep nested outline and this plugin is essentially useless. I'm not sure why, but I am using Atom Light for my theme and the plugin's outline starts dark (#7a7a7a background for first & second tiers), then gets much worse (first #494949 for the third tier, then #2c2c2c for the current position) - all while using #444444 for the text.

<img width="199" alt="screen shot 2017-10-30 at 9 49 06 am" src="https://user-images.githubusercontent.com/11085913/32183853-a767fa16-bd57-11e7-85af-fc5057b657d2.png">

**Comment by scokobro** [comment]
Same here. I'm using a dark theme and my temporary solution is to change line 70 of 'document-outline/styles/document-outline.less' to ...
`      background: fade(@background-color-highlight, 10%);`

and reload the window. Results in the following more usable look...

<img width="298" alt="screen shot 2017-11-24 at 19 45 10" src="https://user-images.githubusercontent.com/1475484/33225984-3bf2dea6-d150-11e7-9086-00580faf4944.png">

Not ideal but makes the outline readable again.

**Comment by mangecoeur** [comment]
I've changed the outline to follow the syntax theme instead of the UI theme, which seems to help it work better with a range of dark/light themes. I can't support every case because fixing one theme breaks others, but this should at least work better for the atom One Light/Dark syntax UI themes in any combination (light ui and dark text, dark ui light text, etc...). I also added a highlight to the currently active heading. Closing this as a result.
